tests/schema/pass/path-item-object-example.yaml

```
@@ -32,4 +32,24 @@ paths:
           type: array
           items:
             type: string
-        style: simple
\ No newline at end of file
+        style: simple
+    additionalOperations:
+      query:
```

```suggestion
      QUERY:
```
We want the original spelling.